modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -735,6 +735,34 @@
 				[b]Note:[/b] As annotations describe their subject, the [annotation @tool] annotation must be placed before the class definition and inheritance.
 			</description>
 		</annotation>
+		<annotation name="@tool_button">
+			<return type="void" />
+			<param index="0" name="text" type="String" default="&quot;&quot;" />
+			<param index="1" name="icon" type="StringName" default="&quot;&quot;" />
+			<description>
+				Mark a function to be displayed as a clickable button in the inspector when running the editor. The parameters are optional and can be used to customize the button.
+				If [param text] is specified it is used as the label text of the button. If [param text] is unspecified then the label text will be derived from the name of the annotated function and automatically capitalized.
+				If [param icon] is specified it is used to fetch a suitable icon texture for the button via [method Control.get_theme_icon] by selecting the desired [param icon] from the [code]"EditorIcons"[/code] theme type. If [param icon] is unspecified the default [code]"Callable"[/code] icon will be used instead.
```

Comment:
It would be nice to find a way to document the `EditorIcons`theme type but that's beyond the scope of this PR.
```suggestion
				If [param icon] is specified, it is used to fetch an icon for the button via [method Control.get_theme_icon], from the [code]"EditorIcons"[/code] theme type. If [param icon] is omitted, the default [code]"Callable"[/code] icon is used instead.
```
or
```suggestion
				[param icon] is used to fetch an icon for the button via [method Control.get_theme_icon], from the [code]"EditorIcons"[/code] theme type. If [param icon] is omitted, the default [code]"Callable"[/code] icon is used instead.
```
or similar.